modules/screen-activity-feed/index.js

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Text, View, Image, ScrollView, StyleSheet, TouchableHighlight } from "react-native";
+
+const pressed = () => {
+  console.log("pressed");
+};
+
+export const ActivityFeed = () => {
```

Comment:
Please change this to be a default export:
https://crowdbotics.github.io/modules/authoring#screen-modules

Reply:
Resolved.